ci-operator/config/redhat-operator-ecosystem/cvp/redhat-operator-ecosystem-cvp-ocp-4.7.yaml

```diff
@@ -11,6 +11,44 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: cvp-common-generic-claim
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.7"
+  steps:
+    dependency_overrides:
+      BUNDLE_IMAGE: brew.registry.redhat.io/rh-osbs-stage/e2e-e2e-test-operator-bundle-container:8.0-3
+      INDEX_IMAGE: brew.registry.redhat.io/rh-osbs-stage/iib:23576
+    env:
+      BASE_DOMAIN: cpaas-ci.devcluster.openshift.com
+      CHANNEL: "4.3"
+      INSTALL_NAMESPACE: '!create'
+      PACKAGE: e2e-test-operator
+      PYXIS_URL: ""
+      TARGET_NAMESPACES: '!install'
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: optional-operators-configure-proxy-registry
+    - ref: optional-operators-cvp-common-apply-secrets
+    - ref: optional-operators-subscribe
+    test:
+    - ref: optional-operators-cvp-common-scorecard
+    - ref: optional-operators-cvp-common-test
+    - as: claim
+      commands: "printenv KUBECONFIG\nprintenv KUBEADMIN_PASSWORD_FILE\noc get node\noc
+        config view\noc whoami  \n"
+      from: ocp-4.7.0-amd64
```

Comment:
This si supposed to be an image, and there's no `ocp-4.7.0-amd64` image.

Reply:
I'm not sure if I get this part correctly, where do I get the image?